### DISCUSSION TITLE
Loaded and read HFS+ mac hdd last night and now will not

### DISCUSSION THREAD

**Discussion by lazertunes** [unsound/hfsexplorer#20] [General]
Hi Eric - Was stoked last night when program worked so well.  Did 'Load device - autodetect which loaded and made available extraction of files.  When I exited there was option to delete temp files which I did.  Today, when mounting HDD on USB, Win 10 note says:  "USB Device Not Recognized - last device malfunctioned and Windows does not recognize it" (which I expect is normal since it is not FAT nor NTFS volume).  As I recall it was mounted as HardDisk8 - but not today.  Any suggestions?  Need to get at data rather badly.  Thanks much for your time - Mike Williams

**Comment by unsound** [comment]
Hi,

It's hard to know what happened between last night and today, hopefully nothing bad. I would suggest that you try to connect it to another computer to see if it recognizes it. There's unfortunately not much HFSExplorer itself can do if Windows doesn't make the hard drive available to it. It relies on the operating system and its drivers to handle the basics and only takes over once it is available as a logical device that can be read and written.

If it can't be accessed from any computer then it's most likely a hardware error that occurred in between and the only way to get access to the data that case is probably by consulting a data recovery company specializing in extracting data from broken devices.

Best regards,

- Erik

**Reply by lazertunes** [reply]
I really appreciate you quick response, Eric.  I'm using a USB 3.0 to mulit-PCIe adapter - so it is maybe the problem.  Your S/W works exactly as expected - and better - I'll keep you updated - if you're interested.  

Thanks again and all the best - Mike
